simconvoi.h

```diff
@@ -418,7 +418,10 @@ class convoi_t : public sync_steppable, public overtaker_t
 	void set_erstes_letztes();
 
 	// returns the index of the vehikel at position length (16=1 tile)
-	int get_vehicle_at_length(uint16);
+	uint16 get_vehicle_at_length(uint32);
+
+	// returns the total vehicle number of the coupling convoys.
+	uint16 get_total_vehicle_count();
```

Comment:
```suggestion
	// returns the total vehicle number including the coupling convoys.
	uint16 get_total_vehicle_count() const;
```

オブジェクトの状態を変化させない関数には `const` をつけてください。

Reply:
対応しました